lib/google_coordinates.rb

```diff
@@ -0,0 +1,25 @@
+require 'httparty'
+
+class GoogleCoordinates
+  def self.location_info(location)
```

Comment:
You have several places with nested `if`s and `return` from within the `if` clauses. We think this makes following the code harder to follow. This is the most extreme example. REQUEST: Could you implement this same with less branching?